terminal.md

```diff
@@ -0,0 +1,20 @@
+**Try it: cd and cd ..**
+
+1. To go back from work directory to "to_do" type (cd ..)
+
+2. To go back from projects_to_delegate directory to "to_do" type cd .. and once again cd .. or we can combine them and type cd ../.. just one time.
```

Comment:
Correct – `cd ../..` works. Even if you omit the `cd`, your terminal will understand what `..` is. Just a tip.